packages/web-frontend/src/containers/DashboardGroup/index.js

```diff
@@ -19,7 +19,7 @@ import styled from 'styled-components';
 import DashboardItem from '../DashboardItem';
 import StateDashboardItem from '../StateDashboardItem';
 import { DASHBOARD_VIEW_MARGIN } from '../../styles';
-import { getUniqueViewId } from '../../utils/getUniqueViewId';
+import { getUniqueViewId } from '../../utils';
```

Comment:
Is this a good change? When would you want to import from `../../utils/getUniqueViewId`?

Reply:
Definitely, in my opinion! Unfortunately there are no clear checks and guidelines on how to a codebase should properly structure its imports/exports. Perhaps this aspect of JavaScript will mature more in the future.

In my opinion, folders (like `utils`) define "modules", that can expose inner functionality (API) like `getUniqueViewId` through an `index.js` file. This allows safe restructuring of inner functionality (moving functions in files/folders) without disrupting the clients of the API.

The only case I would import directly from a file is in unit tests, since a function may stop being exported in the main `index.js` file, but still used internally in the module which means we would still want to test it.

Reply:
That makes sense. I'll try to follow that in future, at least when it makes sense. 👍